kolibri/plugins/management/assets/test/management.js

```diff
@@ -55,6 +56,27 @@ describe('The management module', () => {
         done();
       });
     });
+
+    describe('a "fetch" action', function () {
+      before(function () {
+        this.xhr = sinon.useFakeXMLHttpRequest();
+        this.requests = [];
+        this.xhr.onCreate = req => {
+          this.requests.push(req);
+        };
+      });
+
+      after(function () {
+        this.xhr.restore();
+      });
+
+      it('that makes 4 requests', function () {
```

Comment:
Does the view actually "have a fetch action" and make the requests? I'd expect that to be decoupled from the view.